src/main/java/com/nike/vault/client/VaultClient.java

```diff
@@ -54,6 +56,10 @@
 
     public static final MediaType DEFAULT_MEDIA_TYPE = MediaType.parse("application/json; charset=utf-8");
 
+    public static final String LOG_HTTP_REQUEST_URL_TMPL = "Request URL: {}";
```

Comment:
These shouldn't be public, should they?

Also, I not sure static final is right for log statements.  I see you are trying to make them consistent but what I would have done is written them inline with the method name within each statement.

e.g. logger.debug("enclosingMethodName() requestUrl: {}"

Also, I would remove the space in strings like "Request URL" so that it is easier to automatically pick up the variable name if someone is using a tool like Splunk.